charts/incubator/invidious/templates/_config.tpl

```diff
@@ -0,0 +1,907 @@
+{{/* Define the configmap */}}
+{{- define "invidious.config" -}}
+
+{{- $configName := printf "%s-invidious-config" (include "tc.common.names.fullname" .) }}
+{{- $configEnvName := printf "%s-invidious-env" (include "tc.common.names.fullname" .) }}
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configName }}
+  labels:
+    {{- include "tc.common.labels" . | nindent 4 }}
+data:
+  config.yml: |
+# version compatible 0.20.1+
+#########################################
+#
+#  Database configuration
+#
+#########################################
+
+##
+## Database configuration with separate parameters.
+## This setting is MANDATORY, unless 'database_url' is used.
+##
+db:
+  user: "{{ .Values.invidious.db_user }}"
+  password: "{{ .Values.invidious.db_pass }}"
+  host: "{{ .Values.invidious.db_host }}"
+  port: {{ .Values.invidious.db_port }}
+  dbname: "{{ .Values.invidious.db_name }}"
+
+##
+## Enable automatic table integrity check. This will create
+## the required tables and columns if anything is missing.
+##
+## Accepted values: true, false
+## Default: false
```

Comment:
Once you finish and copy whatever you want for gui's description, delete comments, as Secrets/ConfigMaps has 1MiB size limit